examples/widgets/textbox.py

```diff
@@ -5,9 +5,13 @@
 
 Allowing text input with the Textbox widget.
 
-You can use the Textbox widget to let users provide any text that needs to be
-displayed, including formulas. You can use a submit button to create plots
-with the given input.
+You can use the Textbox widget to let users interactively provide any text 
+that needs to be displayed, including formulas. You can use a submit button to 
+create plots with the given input.
+
+Note to not get confused with 
```

Comment:
Not a language expert, but this sounds a bit colloquial. Can you find a more succinct and affirmative wording? Maybe @story645 can help?

Reply:
Hmm, so I think it's the "you" that's making it sound informal, which is probably fine. But also I don't think the description is fully describing what's going on in the example: 
```
The Textbox widget lets users interactively provide text input, including formulas.  In this example, the plot is updated using the `.on_submit` method. This method triggers the execution of the `submit` function when the user presses enter in the textbox or leaves the textbox. 

Note:  The `Textbox` widget is different from the following static elements:
:doc:`/tutorials/text/annotations` 
:doc:`/gallery/recipes/placing_text_boxes`.
```
I'd also remove the `that needs to be displayed` bit 'cause it's not clear how or why that's being done.

Reply:
Thank you for the suggestion @story645 Suggestions implemented.

Reply:
I'm not seeing the changes. Are you sure you pushed them?

Reply:
Not yet! Sorry I was making comment as I work them through. Will push soon

Reply:
No problem, I was just a bit confused. :smile: